src/NUnitFramework/framework/Attributes/RetryAttribute.cs

```diff
@@ -22,7 +22,7 @@
 // ***********************************************************************
 
 // TODO: Rework this
-// RepeatAttribute should either
+// RetryAttribute should either
 //  1) Apply at load time to create the exact number of tests, or
 //  2) Apply at run time, generating tests or results dynamically
```

Comment:
I think that we should delete all the comments regarding the TODO. As far as I can tell the comment is about `RepeatAttribute` and not `RetryAttribute`, e.g. we cannot at load time calculate the exact number of tests.